### PR TITLE
anchoring bias function that passes in area the person focus too much…

### PR DESCRIPTION
**Pull request by RachelZhong98**
… on and an array of all areas the person has not focused on.

### CODE REVIEW

File: anchoringbias.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    if len(dict2) == 0:
+        return "You have focused on location, person, and time. Do you have other aspects you'd like to focus on?";
+    newFocus = dict2[random.randrange(0, len(dict2))];
+    print(dict2);
+
+    return "You are focusing on " + focus + " too much." + " Do you have any hypothesis regarding " + newFocus;
+
+
+
+dict1 = ["location", "person", "time"];
```
Comment (Stephanie-A) [inline anchoringbias.py]: You can delete these statements, they're just for debugging purposes

File: anchoringbias.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import random
+def anchoringbias(focus, dict1): # pass in the area of focus
+    dict2 = [];
+    for area in dict1:
+        if area != focus:
+            dict2.append(area);
+    if len(dict2) == 0:
+        return "You have focused on location, person, and time. Do you have other aspects you'd like to focus on?";
+    newFocus = dict2[random.randrange(0, len(dict2))];
+    print(dict2);
```
Comment (Stephanie-A) [inline anchoringbias.py]: Why are you printing out the dictionary?

**Review by Stephanie-A — CHANGES_REQUESTED** [review]
Pretty good! Just look over my comments

File: anchoringbias.py (hunk 2 above)
Comment (RachelZhong98) [reply anchoringbias.py]: Sorry, you can get rid of the statement. I added it when debugging my code and forgot to delete it.

Comment (Stephanie-A) [reply anchoringbias.py]: Nah you're all good. You can just edit the files, commit them, and push

**Pull request merged**